src/test/java/step3/domain/LottoNumberTest.java

```diff
@@ -0,0 +1,24 @@
+package step3.domain;
+
+import static org.assertj.core.api.Assertions.*;
+
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+public class LottoNumberTest {
+    @Test
+    void 로또번호_생성() {
+        Assertions.assertThat(LottoNumber.of(1)).isEqualTo(LottoNumber.of(1));
+    }
+
+    @ParameterizedTest(name = "invalid_로또번호_생성")
+    @ValueSource(ints = {-1, 100, 0})
+    void invalid_로또번호_범위() {
```

Comment:
테스트하고자 하는 범위의 경계값을 사용해 주면 어떨까요?
```suggestion
    @ParameterizedTest(name = "invalid_로또번호_생성")
    @ValueSource(ints = {0, 46})
    void invalid_로또번호_범위() {
```

Reply:
실제 오류는 말씀하신 경계값에서 많이 겪는 것 같습니다.
테스트 값에도 더 고민해야 할 것 같습니다. 
수정하겠습니다. 😄